src/redux/actions/samples/createSample.js

```diff
@@ -88,10 +84,8 @@ const createSample = (
 
     pushNotificationMessage('error', message);
 
-    return Promise.reject(message);
```

Comment:
This does not throw an error, so the subsequent line in the enclosing function continues to run